codegen/templates/init_clients.tmpl

```diff
@@ -23,7 +23,7 @@ type Clients struct {
 func CreateClients(
 	config *zanzibar.StaticConfig,
 	gateway *zanzibar.Gateway,
-) *Clients {
+) interface{} {
```

Comment:
Agreed, I think we need to change the generated clients struct and then we can have an interface that is generic and shared between cleints and benchmarkgateway, but we can do that in the future.